ls1intum/Artemis
# Add buttons to Programming Exercises Details Page

### Checklist

### Motivation and Context
Resolves #2468 

### Description
I keep the existing buttons (except the back button) and add all possible buttons that are related to programming exercise to the page. I moved all the buttons to the top of the page.

### Steps for Testing
1. Go to a Programming Exercise Detail page (For Example: Course management -> Exercises -> Details )
2. Check that all the buttons in the page still working.

### Screenshots

Old:

![Capture](https://user-images.githubusercontent.com/36769488/100553912-51dcbe80-3291-11eb-81e5-4831fcd0103f.PNG)
![Capture](https://user-images.githubusercontent.com/36769488/100553921-6456f800-3291-11eb-84fa-9147db713a10.PNG)

New:

![Capture](https://user-images.githubusercontent.com/36769488/100553938-7c2e7c00-3291-11eb-8032-dfd13fc1ae0f.PNG)
